NEWS

```diff
@@ -1,3 +1,10 @@
+06-Dec-2018 IGNITION v0.29.1
+
+  Bug Fixes:
+
+    - Do not follow symlinks if they are the last path element when writing
```

Comment:
"if they are the last path element when writing" seems a bit awkward.  "When writing files, directories, or links, do not follow symlinks if they are the last path element"?